cmd/vic-machine/common/proxy_test.go

```diff
@@ -39,21 +40,29 @@ func TestProcessProxies(t *testing.T) {
 		"httpd://example.com",
 	}
 
+	gnproxies := [...]string{
+		"*",
+		"localhost, .example.com",
```

Comment:
Maybe we should also include a test case without a space. (E.g., "localhost,.example.com")

Reply:
Done